src/py/mat3ra/notebooks_utils/material.py

```
@@ -0,0 +1,163 @@
+                    )
+                    continue
+                name = os.path.splitext(filename)[0]
+                log(f"{index}: {name}", SeverityLevelEnum.INFO, force_verbose=verbose)
+                index += 1
+                data_from_host.append(data)
+    except FileNotFoundError:
+        log(f"No data found in the '{folder_path}' folder.", SeverityLevelEnum.ERROR, force_verbose=verbose)
+        return []
+    try:
+        materials = [MaterialWithBuildMetadata.create(item) for item in data_from_host]
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)